### ISSUE TITLE
Change the `status` key to `legacy_status` in `\users\:userName` endpoint 

### ISSUE THREAD

**Issue by heyrandhir** [Real-Dev-Squad/website-backend#789]
Currently, in the `\users\:userName` endpoint we get the object which has a key `status` and its value is one of the items in this list [ `ooo`, `active`, `idle`]. We are also creating a new `Status collection`. Hence we need to change the `status` key in the `\users\:userName` endpoint to `legacy_status`. Also wherever in our apps we are using `status` it needs to be updated to `legacy_status` so that our Apps don't break because of this change. 

TLDR:

- Change `status` key to `legacy_status` in `\users\:userName` endpoint.
- Update the `status` key to `legacy_status` in all the apps.
